CONTRIBUTING.md

```diff
@@ -87,6 +87,18 @@ changes.
 > confusing to IDEs (or the `watch:docs` and `watch:js` tasks) that might be performing
 > their own watching, or run up against file system limits.
 
+#### `doit serve:*`
+
+A number of development servers can be started, each on separate ports:
+
+- core assets from `./app`:
+  - `doit serve:js`
+  - `doit serve:py`
+- example site it `./build/docs-app`:
+  - `doit serve:docs:app`
+- jupyterlab
+  - `doit serve:lab`
```

Comment:
Just curious: do we expect contributors to use sock jupyterlab, or is it for convenience?

Reply:
yeah, that and authoring "heavy" docs notebooks like `cli.ipynb`... will add a note above

Reply:
indeed, as a separate item, we should add `jupyterlab-myst`

Reply:
6e12cbe

Reply:
Alright sounds good!